AsyncDisplayKitTests/ArrayDiffingTests.m

```diff
@@ -10,13 +10,58 @@
 
 #import "NSArray+Diffing.h"
 
+@interface NSArray (ArrayDiffingTests)
+- (NSIndexSet *)_asdk_commonIndexesWithArray:(NSArray *)array compareBlock:(BOOL (^)(id lhs, id rhs))comparison;
```

Comment:
@maicki @levi I know there are other areas where we test internal methods. I think it depends mainly on whether they have a contained set of expected behaviors, for example if they are a functional style and algorithmic like this case, I don't see a disadvantage in having tests. If the tests ever break, it will be a reminder that the behavior has changed in a material way, and if that is intended then we can simply delete the tests.

I don't feel strongly about this as a general point, and am both interested and open to the community refining are testing practices in this respect or others. Just mainly for this specific case, it seems like the tests have value at the moment, whereas we can always delete them later if they no longer have value.